snooty/test_parser.py

```diff
@@ -847,3 +847,30 @@ def test_cardgroup() -> None:
         </directive>
         </root>""",
     )
+
+
+def test_toctree() -> None:
```

Comment:
I'd love if we also had another test for the case of a URL with no title

Reply:
This case should fail, correct?

Reply:
We would have an assertion that a correct diagnostic is assigned to the page